alchemist/alchemist-swingui/src/main/kotlin/it/unibo/alchemist/boundary/monitors/AbstractFXDisplay.kt

```diff
@@ -41,6 +42,10 @@ import java.util.stream.Stream
  * @param <T> The type which describes the [Concentration] of a molecule
  * @param <P> The type of position
  */
+@SuppressFBWarnings(
+    "NP_NONNULL_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR",
```

Comment:
Looks like a false positive for Kotlin. We may want to disable the check for every kt source.

Reply:
I don't know if it's really a false positive for Kotlin - here, the `wormhole` field is initialized in a `protected open fun init(environment: Environment<T, P>): Unit` function which, for this class, is always called before using the wormhole; this might not always be the case for other Kotlin classes though.

Reply:
got it. Suppression is tolerable then.